src/nvim/vterm/state.c

```diff
@@ -17,6 +17,11 @@
 
 #define strneq(a, b, n) (strncmp(a, b, n) == 0)
 
+// Primary Device Attributes (DA1) response.
+// We make this a global (extern) variable so that we can override it with FFI
+// in tests.
+char vterm_primary_device_attr[] = "1;2;52";
```

Comment:
FYI, the `1;2` response is a device-specific format that predates DEC's standardization of the `DA` response. The second parameter is actually a bitfield value that represents extensions specific to the VT100 range of devices. It isn't expected to have additional parameters after it.

If you want to indicate a level-1 device using the _standard_ `DA` format, the first parameter should be `61`, and then any additional parameters represent the supported extensions. For example, a level-1 device with just the new clipboard extension would be `61;52`.

Probably not super important, though, since many apps that parse the `DA` response will likely ignore the first parameter anyway.